docs/pages/documentation/10-resources/faq.md

```diff
@@ -28,31 +28,14 @@ There are lots of ways you can get involved! Please take a look at our [contribu
 
 ## Bugs and strange behaviour
 
-### Why does Grakn hang when I try to start it?   
-
-I am running `grakn.sh start` but it hangs on `Starting Cassandra`. Why?
-
-This may be because you have cloned the Grakn repo into a directory which has a space in its name (e.g. `/grakn test`). You can build our code successfully, but when you start `grakn.sh`, it hangs because Cassandra needs you to have single word pathnames. Remove the spaces (e.g. `/grakn_test`) and try again.
-
-There are other possible reasons why Grakn hangs starting Cassandra. One may be that some other application is using the port 7199, which Cassandra needs.  To find out what is using port 7199:
-`lsof -i tcp:7199`
-
-From there, you'll see the PID of application using that port. Check if you can safely kill it or change its port. It may be that another instance of Cassandra is blocking it, and you can simply kill it using:
-`pkill -9 java`
-
-Then try `grakn.sh start` again.
-
-Failing that, you can often find out more information by looking in the `/logs` directory under your Grakn installation.  
-
-Please see the answer to the question below "Can I run Grakn on an existing Cassandra Platform?" if you are already using Cassandra and want to run Grakn on a different instance of Cassandra to our default.
-
 ### Why am I getting ghost vertices?
 
-In a  transaction based environment it is possible to have one transaction removing a concept while another concurrently modifies the same concept. Both
-transactions may successfully commit if the backend is eventually consistent, e.g. [Janus Cassandra](http://docs.janusgraph.org/latest/common-questions.html).
+In a  transaction based environment it is possible to have one transaction removing a concept while another concurrently modifies the same concept. 
+Both transactions may successfully commit in eventually consistent environment.
```

Comment:
"in _an_ eventually consistent environment" I guess.

Reply:
"In a  transaction" has double whitespace